src/BuiltinExtensions/ComfyUIBackend/WorkflowGeneratorSteps.cs

```diff
@@ -976,6 +977,14 @@ JArray doMaskShrinkApply(WorkflowGenerator g, JArray imgIn)
                 {
                     PromptRegion.Part part = parts[i];
                     string segmentNode;
+                    bool sharper = false;
+
+                    if (part.DataText.StartsWith("sharp-"))
```

Comment:
this is also a full separate new feature addition that should be done separately